spec/dummy/config/initializers/serialize_positional_to_kwargs_bridge.rb

```diff
@@ -0,0 +1,19 @@
+module Dummy
```

Comment:
This feels like it should be part of the plugin itself, and not dummy.  The dummy app is only for tests here, so I presume we will hit this same issue in application code where the dummy won't be present.  Perhaps it could be done similar to how we [patch migrate for clearing caches](https://github.com/ManageIQ/manageiq-schema/blob/master/lib/manageiq/schema/migrate_with_cleared_schema_cache.rb).

Reply:
Nice, I can look into that.  Good idea.